lib/dns.js

```diff
@@ -38,6 +37,9 @@ function errnoException(err, syscall, hostname) {
   return ex;
 }
 
+function validIpVersion(isIpResult) {
+  return isIpResult !== 0;
```

Comment:
@mscdex I would agree- but then it would certainly need a comment to explain it. I would not think "validIpVersion" from `isIpResult !== 0`